bot/client.js

```diff
@@ -11,11 +11,37 @@ const client = new Discord.Client();
 // Pre-load controllers
 const controllers = require('./controllers')();
 
+// Function to add points for chatting
+const awardPointsforChatting = async (message) => {
+  const { content, channel, author } = message;
+  if (channel.type !== 'dm' && content.length >= 5) {
+    const messagesPoints = 0.2;
+    // SQL select statement
+    const memberData = await Member.findAll({
+      attributes: ['messagesCount', 'points', 'verified'],
+      where: { discordUser: author.id },
+    });
+    await util.log('Member data from SQL call', memberData[0].dataValues, 4);
+    let { messagesCount, points } = memberData[0].dataValues;
+    const { verified } = memberData[0].dataValues;
+    messagesCount += 1;
+    points += messagesPoints;
+    if (verified) {
+      await Member.update(
+        { messagesCount, points: parseFloat(points.toFixed(2)) },
+        { where: { discordUser: author.id } },
+      ).then((updatedRows) => {
+        util.log('Updated Result', updatedRows, 4);
+      });
+    }
+  }
+};
+
 // Award bonus points when user reaches every 1000 messages
 const awardBonusPoints = async (user) => {
   const amountOfBonusPoints = 100;
   // Get User Message Count
-  const memberData = await models.Member.findAll(
+  const memberData = await Member.findAll(
```

Comment:
I like how you deconstructed the model, makes it a lot cleaner. Looks good to me!